phase2/src/tests/test_issues.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+import json
+from ..jira import Jira
+from .fixtures.issue_info import IssueInfo, prep_issue
+
+
+# issue tests: create/update/delete
+class TestIssue:
+
+    @classmethod
+    def teardown_class(cls):
+        jira = Jira()
+        jira.authenticate("Alexander_Artemov", "Alexander_Artemov")
+        jira.cleanup()
+
+    @pytest.mark.parametrize("issue_data", [IssueInfo(project="AQAPYTHON", description="this is a test issue. 01", type="Bug", expected=400),
+                                            IssueInfo("AQAPYTHON", "", "this is issue with empty summary. 02", "Bug", 400),
+                                            IssueInfo("AQAPYTHON", "AA001 - test issue test issue test issue test issue test issue test issue test issue test issue test issue test issue test issue test issue test issue test issue test issue test issue test issue test issue test issue test issue test issue test issue test issue test issue test issue test issue test issue test issue test issue test issue 02", "this is an issue with looooong text in summary. 03", "Bug", 400),
```

Comment:
Looks good.
For better readability of the test I would suggest to move this long name to a static method, which will return same text.